### ISSUE TITLE
Bug - Failed to run tests

### ISSUE THREAD

**Issue by rohlacanna** [openssl/openssl#20301]
Hi everyone, everything fine?
I am running the following command:
```bash
sudo make test
```
and i am having these glitches how can i solve this?

```bash
Test Summary Report
-------------------
../test/recipes/80-test_ssl_new.t                (Wstat: 256 Tests: 29 Failed: 1)
  Failed test:  12
  Non-zero exit status: 1
Files=158, Tests=2640, 104 wallclock secs ( 2.14 usr  0.28 sys + 94.47 cusr 20.53 csys = 117.42 CPU)
Result: FAIL
make[1]: *** [Makefile:208: _tests] Error 1
make[1]: Leaving directory '/usr/local/src/openssl-1.1.1m'
make: *** [Makefile:205: tests] Erro 2
```

**Comment by paulidale** [comment]
Try `make test VERBOSE=1` to get proper output.

**Comment by anchorbob** [comment]
I also ran into similar fails. Adding VERBOSE=1 in "make test " does not provide extra error message:
`Test Summary Report
-------------------
../test/recipes/80-test_ssl_new.t                (Wstat: 256 Tests: 29 Failed: 1)
  Failed test:  12
  Non-zero exit status: 1
Files=155, Tests=1264, 42 wallclock secs ( 0.70 usr  0.22 sys + 47.26 cusr  4.66 csys = 52.84 CPU)
Result: FAIL
make[1]: *** [Makefile:208: _tests] Error 1
make[1]: Leaving directory '/home/user/openssl-1.1.1g'
make: *** [Makefile:205: tests] Error 2
`

**Comment by paulidale** [comment]
That's the end of the output.  Look back to where the SSL new test was run.  There should be more information there.

**Comment by mattcaswell** [comment]
To just run the ssl_new test you can do this `make VERBOSE=1 TESTS=test_ssl_new test`

**Comment by rohlacanna** [comment]
> That's the end of the output. Look back to where the SSL new test was run. There should be more information there.

When running this command it returns me: `make: Nothing to be done for 'test'.`

**Comment by nhorman** [comment]
marking as inactive, will be closed at the completion of 3.4 dev without further input
